pkgs/desktops/elementary/pantheon-agent-geoclue2/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchFromGitHub, elementary, pkgconfig, meson, ninja, vala, glib, gtk3
+, libgee, desktop-file-utils, dbus-glib, geoclue2, gobjectIntrospection, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "pantheon-agent-geoclue2";
+  version = "1.0";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "elementary";
+    repo = pname;
+    rev = version;
+    sha256 = "0m7wbyqqivfwpd2m4jcxg4vl3vmfxkj7285h283p9m6nfh9vz960";
+  };
+
+  passthru = {
+    updateScript = elementary.updateScript {
+      repoName = pname;
+    };
+  };
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    gobjectIntrospection
+    meson
+    ninja
+    pkgconfig
+    vala
+    wrapGAppsHook
+   ];
+
+  buildInputs = [
+    dbus-glib
+    geoclue2
+    glib
+    gtk3
+    libgee
+   ];
+
+  # This should be provided by a post_install.py script
+  postInstall = ''
```

Comment:
Could you add a link to https://github.com/elementary/pantheon-agent-geoclue2/pull/21